interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is defined when a function is being written, and determines what arguments will be required. An argument itself is the actual value being fed into a function when it is called.
 
-Researched answer:
+Researched answer: Specifically, the argument is provided into the function invocation, between the parentheses. It will then be substituted into the function, wherever the corrosponding parameter is used. So, if a function has a paramter called string, and that function is called with the argument "hello", "hello" will be used wherever the parameter 'string' is used.
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: .map takes in another function as a parameter, but is used on an array. So, I think that the only parameter is the function, and that it is required.
 
-Researched answer:
+Researched answer: The .map function takes in a function as parameter, but that function also has its own parameters, currentValue, index, and array, so they can be though of as sub-parameters.
```

Comment:
Good research! .map() does take the parameters of current value, the index and the array. However, remember the only required parameter is the value.